generic/build

```diff
@@ -30,23 +30,20 @@
 set -e
 
 die() {
-	local m="$1"
```

Comment:
The removal of reassigning positional parameters into local-scope variables using "local" should be safe and makes the code a bit more readable. From Dash man-page:

_"When a shell function is executed, all of the shell positional parameters (except $0, which remains unchanged) are set to the arguments of the shell function.  The variables which are explicitly placed in the environment of the command (by placing assignments to them before the function name) are made local to the function and are set to the values given.  Then the command given in the function definition is executed.  The positional parameters are restored to their original values when the command completes."_

From Bash man-page:

_"The positional parameters are temporarily replaced when a shell function is executed (see FUNCTIONS below)."_

So there should be no risk of leaking the positional parameters from the function.